vim/yanking-text-to-register.md

```diff
@@ -0,0 +1,21 @@
+# Yanking text to register
+
+Yanking and deleting text will save the text to the same unnamed register. Which means every time you delete a word, your 
```

Comment:
I just chose "Every time you delete...". Fixed in commit 4978896c50842e90dd6e546989ce91da16201049. Thanks!